src/renderer/components/virtual-list/virtual-list.tsx

```diff
@@ -4,24 +4,26 @@ import "./virtual-list.scss";
 
 import React, { Component } from "react";
 import { observer } from "mobx-react";
-import { ListChildComponentProps, VariableSizeList } from "react-window";
-import { cssNames } from "../../utils";
+import { Align, ListChildComponentProps, ListOnScrollProps, VariableSizeList } from "react-window";
+import { cssNames, noop } from "../../utils";
 import { TableRowProps } from "../table/table-row";
 import { ItemObject } from "../../item.store";
 import throttle from "lodash/throttle";
 import debounce from "lodash/debounce";
 import isEqual from "lodash/isEqual";
 import ResizeSensor from "css-element-queries/src/ResizeSensor";
 
-interface Props {
-  items: ItemObject[];
+interface Props<T extends ItemObject = any> {
```

Comment:
Does defaulting this to `any` gain us anything. I believe that this is a code smell.

Reply:
It does, since `ItemObject` contains 2 specific methods `getId()` & `getName()` which is related to workspaces. But from now on we're using virtual list for any kind of lists, so this line helps to avoid typescript errors.